src/Basque/Mews.Fiscalizations.Basque/DtoToModelConverter.cs

```diff
@@ -28,11 +29,45 @@ internal static class DtoToModelConverter
         );
     }
 
+    public static SendInvoiceResponse Convert(
+        LROEPJ240FacturasEmitidasConSGAltaRespuesta response,
+        string qrCodeUri,
+        string xmlRequestContent,
+        string xmlResponseContent,
+        String1To100 signatureValue
+        )
+    {
+        var invoiceIdentifier = response.Registros.Single().Identificador.Item as IDFacturaType;
+        var invoiceState = response.Registros.Single().SituacionRegistro;
+        return new SendInvoiceResponse(
+            xmlRequestContent: xmlRequestContent,
+            xmlResponseContent: xmlResponseContent,
+            qrCodeUri: qrCodeUri,
+            tbaiIdentifier: invoiceIdentifier.NumFactura,
+            received: DateTime.ParseExact(invoiceIdentifier.FechaExpedicionFactura, "dd-MM-yyyy", CultureInfo.InvariantCulture),
+            state: ConvertBizkaiaState(invoiceState.EstadoRegistro),
+            description: invoiceState.DescripcionErrorRegistroES,
+            stateExplanation: string.Empty,
```

Comment:
Let's not send empty strings, if the 2 models are completely different, then its better to create a new response model for Bizkaia rather than hacking it like this
or if they are very similar, then you can make this nullable rather than sending empty strings